packages/react-devtools-timeline/src/content-views/ComponentMeasuresView.js

```diff
@@ -150,6 +165,15 @@ export class ComponentMeasuresView extends View {
           break;
       }
     }
+
+    const searchRegExp = this._viewState.searchRegExp;
+    const isSearchMatch =
+      searchRegExp !== null &&
+      componentMeasure.componentName.match(searchRegExp);
```

Comment:
I wonder if we should cache this result to avoid re-evaluating every pan/zoom? Something like...

```js
if (this._cachedSearchRegExp !== searchRegExp) {
  this._cachedSearchRegExp = searchRegExp;
  this._cachedIsSearchMatch =
    searchRegExp !== null && componentMeasure.componentName.match(searchRegExp);
}

const isSearchMatch = this._cachedIsSearchMatch;
```